spaceship/lib/spaceship/tunes/display_family.rb

```diff
@@ -0,0 +1,66 @@
+require_relative "../module"
+
+module Spaceship
+  module Tunes
+    # A definition of differnet styled displays used by devices
+    # that App Store Connect supports storing screenshots.
+    # Display styles often only vary based on screen resolution
+    # however other aspects of a displays physical apperance are
+    # also factored (i.e if the home indicator is provided via
+    # a hardware button or a software interface).
+    class DisplayFamily
+      # The display family name from the App Store Connect API that
+      # is used to assign `Spaceship::Tunes::AppScreenshot` objects
+      # to the correct display families via its `device_type` property.
```

Comment:
`device_type` of `Spaceship::Tunes::AppScreenshot`? This is currently formulated a bit confusing.

Reply:
`DeviceType#exists?` is used twice in Spaceship and once in Deliver and `DeviceType#types` appears to no longer be used (other than for `exists?`) 

I could introduce something like the following to `DisplayFamily` to replace the current usages:

```diff
- exists = Spaceship::Tunes::DeviceType.exists?(screenshot.device_type)
+ exists = !Spaceship::Tunes::DisplayFamily.for_name(screenshot.device_type.to_sym).nil?
```

Open to suggestions to make that look nicer if you have any 👍

Reply:
Hmmm yeah.. How about this: 

> The display family name from the App Store Connect API that is used when uploading or listing screenshots. This value is then assigned to the `Spaceship::Tunes::AppScreenshot#device_type` attribute.

Maybe it doesn't need the second part but I thought it would be useful to explain where it comes from in the rest of the project?

Reply:
Yeah, context is important here - ever pointer helps :) Your suggestion is much better.